src/Ledger.Net/Constants.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Ledger.Net
 {
-    public class Constants
+    public static class Constants
```

Comment:
Nice

Although, I'm probably going to move the constants to an app specific area, because the constants are different across apps and it would get very confusing to have them all in one spot across apps.